crates/re_types/src/archetypes/image_ext.rs

```diff
@@ -28,7 +28,7 @@ impl Image {
                 assign_if_none(&mut data.shape[non_empty_dim_inds[1]].name, "width");
             }
             3 => match data.shape[non_empty_dim_inds[2]].size {
-                3 | 4 => {
+                1 | 3 | 4 => {
```

Comment:
this dangerous or even wrong: Having 1 here causes out of bound access in the following lines.
Might also be wrong since I think we filter empty dimensions out somewhere?